api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>Similar to my variable-naming comment above: Can do</p><p> </p><p><code>TEMP_METRIC_ID = 2540047</code></p><p><code>ETA_METRIC_ID = 2010042</code></p><p><code>NDVI_METRIC_ID = 430029</code></p><p><code>SOIL_MOISTURE_METRIC_ID = 15531082</code></p><p><code>RAINFALL_METRIC_ID = 2100031</code></p><p> </p><p>Also can consolidate the if/else logic like:</p><p> </p><p><code>&nbsp;&nbsp;if(metric_id==RAINFALL_METRIC_ID):# Trmm</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;metric_name = '7 day cumulative rainfall'</code></p><p><code>&nbsp;&nbsp;elif(metric_id==SOIL_MOISTURE_METRIC_ID):# Soil Moisture</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;metric_name = 'soil moisture'</code></p><p><code>&nbsp;&nbsp;elif(metric_id==NDVI_METRIC_ID):# NDVI</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;metric_name = 'vegetation health (NDVI)'</code></p><p><code>&nbsp;&nbsp;elif(metric_id==ETA_METRIC_ID):# ETA</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;metric_name = 'evapotranspiration'</code></p><p><code>&nbsp;&nbsp;elif(metric_id==TEMP_METRIC_ID):# Temp</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;metric_name = '7 day average temperature'</code></p><p><code>&nbsp;&nbsp;else:</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;return False, ''</code></p><p><code>&nbsp;&nbsp;return True, metric_name</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>yep! thanks </p>